x-pack/plugins/monitoring/public/alerts/status.tsx

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { EuiToolTip, EuiHealth } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+import { CommonAlertStatus } from '../../common/types';
+import { AlertSeverity } from '../../common/enums';
+import { AlertState } from '../../server/alerts/types';
+import { AlertsBadge } from './badge';
+
+interface Props {
+  alerts: { [alertTypeId: string]: CommonAlertStatus };
+  showBadge: boolean;
+  showOnlyCount: boolean;
+}
+export const AlertsStatus: React.FC<Props> = (props: Props) => {
+  const { alerts, showBadge = false, showOnlyCount = false } = props;
+
+  let atLeastOneDanger = false;
+  const count = Object.values(alerts).reduce((cnt, alertStatus) => {
+    if (alertStatus.states.length) {
+      for (const state of alertStatus.states) {
+        if ((state.state as AlertState).ui.severity === AlertSeverity.Danger) {
+          atLeastOneDanger = true;
+        }
+      }
```

Comment:
nit: Just some optimization
```suggestion
      if (!atLeastOneDanger) {
        for (const state of alertStatus.states) {
          if ((state.state as AlertState).ui.severity === AlertSeverity.Danger) {
            atLeastOneDanger = true;
            break;
          }
        }
      }
```

Reply:
looks good!